### PR TITLE
Surveypages

### PR DESCRIPTION
**Pull request by bell-ryan-w**
pages 17-30, and minor admin landing changes

### CODE REVIEW

**Comment by jenn1193** [comment]
Fix merge conflicts

**Comment by jenn1193** [comment]
Where are these pages living?  If they're on the website, they shouldn't need a head in the html.

File: mean/modules/core/client/views/admin_landing.html [hunk 1]
```diff
@@ -94,7 +94,15 @@ <h3 class="modal-title">Create New User</h3>
       <div class="col-xs-6">
         <button class="btn btn-info button_admin" ng-click="">User Portal</button>
       </div>
+<<<<<<< HEAD
```
Comment (jenn1193) [inline mean/modules/core/client/views/admin_landing.html:97]: You need to get rid of the merge conflicts

File: mean/modules/survey/views/p17_30.html [hunk 2]
```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:4]: If these are living in the website, why are you including extra stylesheets?

File: mean/modules/survey/views/p17_30.html [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+    <body>
+        <h3>EXAMINATION</h3>
+        <hr>
+        <table class="info">
+            <tr>
+                <td class="col-md-1">
+                  <h4>RESPIRATORY</h4>
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="respiratory_na" />NA or Unable to assess
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:22]: Are you doing anything with the input names?  How are you binding the information?

**Comment by jenn1193** [comment]
Get rid of your merge commit messages.

**Comment by jenn1193** [comment]
How are you binding the information?  How have you tested how the information is getting passed to the backend?

File: mean/modules/survey/views/p17_30.html [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+                  <td class="col-md-1">
+                    Cyanosis
+                  </td>
+                  <td class="col-md-9">
+                    <input type="checkbox" name="cyanosis" />
+                  </td>
+              </tr>
+              <tr>
+                  <td class="col-md-1">
+                    &nbsp &nbsp &nbsp &nbsp Location:
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:59]: What are the &nbsp for?

File: mean/modules/survey/views/p17_30.html [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+                  <td class="col-md-1">
+                    Rhonchi
+                  </td>
+                  <td class="col-md-9">
+                    <input type="checkbox" name="rhonchi" />
+                  </td>
+              </tr>
+              <tr>
+                  <td class="col-md-1">
+                    &nbsp &nbsp &nbsp &nbsp Location:
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:75]: Again, what are the &nbsp for?

File: mean/modules/survey/views/p17_30.html [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+                  </td>
+                  <td class="col-md-9">
+                    <input type="checkbox" name="tracheostomy" />
+                  </td>
+              </tr>
+            </table>
+            <table class="info">
+                <tr>
+                    <td class="col-md-1">
+                      <h4>CARDIOVASCULAR</h4>
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:141]: Inconsistant naming/titling

File: mean/modules/survey/views/p17_30.html [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+        <table class="info">
+            <tr>
+                <td class="col-md-1">
+                  <h4>RESPIRATORY</h4>
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="respiratory_na" />NA or Unable to assess
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="respiratory_abnormal" />Abnormal
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:25]: Don't need to close input tags.  What is the "Abnormal" string for?

File: mean/modules/survey/views/p17_30.html [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+                      <tr>
+                          <td class="col-md-1">
+                            PEG Tube
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="pegtube" />
+                          </td>
+                      </tr>
+                    </table>
+    <!-- Insert pagination here -->
```
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:389]: How is this getting inserted?  You can't just say insert here.

File: mean/modules/survey/views/p18_30.html [hunk 9]
```diff
@@ -0,0 +1,478 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```
Comment (jenn1193) [inline mean/modules/survey/views/p18_30.html:5]: Again, where is this living?  Why are you including extra stylesheets?

File: mean/modules/survey/views/p20_30.html [hunk 10]
```diff
@@ -0,0 +1,268 @@
...
+            </tr>
+            <tr>
+                <td class="col-md-1">
+                  <p>Angina - Stable</p>
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="angina" />
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="angina_stable" />Stable <br/>
```
Comment (jenn1193) [inline mean/modules/survey/views/p20_30.html:43]: If there's a break line between every one of these, first, it's an incorrect tag format, second, create a class for this and put the styling in the css

File: mean/modules/survey/views/p22_30.html [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+
+  <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="https://code.angularjs.org/1.4.3/angular.js"></script>
+    <link rel="stylesheet" href="survey.css" />
+    <script src="gen%20info%20JS.js"></script>
+  </head>
+
+<!-- Insert header here -->
```
Comment (jenn1193) [inline mean/modules/survey/views/p22_30.html:12]: ???

File: mean/modules/survey/views/p17_30.html (hunk 8 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:389]: This was carried over from the example page that I used to generate. I don't know what it is for but I left it as a reference for whoever was using it.

File: mean/modules/survey/views/p17_30.html (hunk 4 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:59]: &nbsp is html version of "tab" it is indent the text on the survey page

File: mean/modules/survey/views/p17_30.html (hunk 5 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:75]: again, "tab"

File: mean/modules/survey/views/p17_30.html (hunk 6 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:141]: I don't know what you mean

File: mean/modules/survey/views/p17_30.html (hunk 7 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:25]: Abnormal is the visual text that shows up on the page next to the checkbox

File: mean/modules/survey/views/p18_30.html (hunk 9 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p18_30.html:5]: again I was just generating pages to help get this done, I thought all of this was part of the standard includes for the pages

Comment (bell-ryan-w) [inline mean/modules/survey/views/p18_30.html:5]: should I just delete this?

File: mean/modules/survey/views/p17_30.html (hunk 2 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:4]: I used a template to generate pages. Basically help complete the grunt work behind 30 pages. Should I delete ?

File: mean/modules/survey/views/p17_30.html (hunk 3 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:22]: same as above these are just suppose to be pages 17-30 to help relieve stress on other members

Comment (bell-ryan-w) [inline mean/modules/survey/views/p17_30.html:22]: if I need to change the format tell me how i can help...

File: mean/modules/survey/views/p20_30.html (hunk 10 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p20_30.html:43]: We have 1 day and some change... I can try to figure out how to do this but I'm not sure if it is worth the time?

File: mean/modules/survey/views/p22_30.html (hunk 11 above)
Comment (bell-ryan-w) [inline mean/modules/survey/views/p22_30.html:12]: ???

File: mean/modules/core/client/views/admin_landing.html (hunk 1 above)
Comment (bell-ryan-w) [inline mean/modules/core/client/views/admin_landing.html:97]: Deleted

File: mean/modules/survey/views/p17_30.html (hunk 2 above)
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:4]: Yes

File: mean/modules/survey/views/p17_30.html (hunk 3 above)
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:22]: You didn't answer the question.

File: mean/modules/survey/views/p17_30.html (hunk 7 above)
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:25]: Should that be in a label then?

File: mean/modules/survey/views/p17_30.html (hunk 6 above)
Comment (jenn1193) [inline mean/modules/survey/views/p17_30.html:141]: Why is this all capitalized while the other sections are not.

File: mean/modules/survey/views/p18_30.html (hunk 9 above)
Comment (jenn1193) [inline mean/modules/survey/views/p18_30.html:5]: Please refer to other pages in the app for consistency.

File: mean/modules/survey/views/p20_30.html (hunk 10 above)
Comment (jenn1193) [inline mean/modules/survey/views/p20_30.html:43]: It takes two minutes.

File: mean/modules/survey/views/p22_30.html (hunk 11 above)
Comment (jenn1193) [inline mean/modules/survey/views/p22_30.html:12]: Why is there an insert header here?  Please refer to the other pages in the app for consistency and use them as your guideline.

File: mean/modules/core/client/views/admin_landing.html (hunk 1 above)
Comment (jenn1193) [inline mean/modules/core/client/views/admin_landing.html:97]: Then what is this?

**Pull request closed without merging**